includes/embeds/class-amp-twitter-embed-handler.php

```diff
@@ -64,11 +64,31 @@ public function unregister_embed() {
 	 *
 	 * @see \WP_Embed::shortcode()
 	 *
+	 * @param array $matches URL pattern matches.
 	 * @return string Rendered oEmbed.
 	 */
-	public function oembed() {
+	public function oembed( $matches ) {
 		_deprecated_function( __METHOD__, '1.1' );
-		return '';
+		$id = false;
+
+		if ( isset( $matches['tweet'] ) && is_numeric( $matches['tweet'] ) ) {
+			$id = $matches['tweet'];
+		}
+
+		if ( ! $id ) {
+			return '';
+		}
+
+		$this->did_convert_elements = true;
+		return AMP_HTML_Utils::build_tag(
+			$this->amp_tag,
+			[
+				'data-tweetid' => $id,
+				'layout'       => 'responsive',
+				'width'        => $this->args['width'],
+				'height'       => $this->args['height'],
+			]
+		);
```

Comment:
I don't think we need this method anymore, do we? Can't it just be emptied out, as I did in: e168813e521d0138383499a84632bc8637ae6727

```php
	/**
	 * Render oEmbed.
	 *
	 * @deprecated Since 1.1 as now the sanitize_raw_embeds() is used exclusively, allowing the
	 *             original oEmbed response to be wrapped with `amp-twitter`.
	 *
	 * @see \WP_Embed::shortcode()
	 *
	 * @return string Rendered oEmbed.
	 */
	public function oembed() {
		_deprecated_function( __METHOD__, '1.1' );
		return '';
	}
```

Reply:
Actually, we might as well just remove it entirely, and it is not being used. It has been deprecated since 1.1.

Reply:
Good point, it's now removed in https://github.com/ampproject/amp-wp/pull/3678/commits/6ae3419ae2d2c7aae7f93fee751d4f368441b115.